apis/apiextensions/v1/composition_transforms_test.go

```diff
@@ -248,6 +248,26 @@ func TestStringResolve(t *testing.T) {
 				o: "crossplane",
 			},
 		},
+		"ConvertToBase64": {
+			args: args{
+				stype:   StringTransformConvert,
+				convert: &tobase64,
+				i:       "CrossPlane",
+			},
+			want: want{
+				o: "Q3Jvc3NQbGFuZQ==",
+			},
+		},
+		"ConvertFromBase64": {
+			args: args{
+				stype:   StringTransformConvert,
+				convert: &frombase64,
+				i:       "Q3Jvc3NQbGFuZQ==",
+			},
+			want: want{
+				o: "CrossPlane",
+			},
+		},
```

Comment:
Could you add a test for the error case here (i.e. malformed base64 input) please.

Reply:
Done